docs/tutorial/planning.txt

```diff
@@ -0,0 +1,82 @@
+My current plan for my COMBINE 25 tutorial (that will then be used for the spras doc tutorials)
+- I will be testing this on a user that has basic python knowledge but doesn't know spras
+
+*Tony will be giving a presentation on PRAs and SPRAS prior (hopefully) to this tutorial
+
+1) basic
+Goal: Get new users comfortable installing SPRAS
+- don't need to preinstall conda, docker, vscode(?) (or run on terminal), and cytoscape prior, but it is recommended to to make it easier/faster
+- Basic Python knowledge (running scripts, installing packages)
+Goal: Run SPRAS with one algorthm on a small example dataset, then re-run it with three different parameter settings to see how network structure changes.
+
+Slides and/or Information to add to docs directly:
+Overview of PRAs (Pathway Reconstruction Algorithms) in plain language
+What SPRAS is and what problems it solves
+Installation & environment setup (Docker Anaconda and Cytoscpare locally)
+SPRAS directory structure for a user & configuration files
+- config folder
+- input folder
+- output folder
+    - can control the creation of subfolders of outputs in the config file
+    - might need to be in a seperate step/slide
+Running SPRAS on a provided example dataset 
+- set up one algortihm and run
+- then run one algortihm with 3 different preset parameter settings
+Understanding the outputs
+- show the output and structure
+- show it visulized
+Viewing logs and monitoring runs
+
+Things to make:
+- need one small dataset to run on (either make a dummy one or just use egfr)
```

Comment:
Building around EGFR will be the least amount of work, and that pathway may be a good fit for the COMBINE audience. They probably model it frequently in that style of systems biology community.